utils/api_config_loader.py

```
@@ -0,0 +1,283 @@
+    
+    Returns:
+        Dict: 配置字典
+        
+    Raises:
+        FileNotFoundError: 配置文件不存在
+        json.JSONDecodeError: JSON格式错误
+    """
+    config_path = _get_config_file_path()
+    
+    if not config_path.exists():
+        raise FileNotFoundError(f"配置文件不存在: {config_path}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)